src/components/v2/SuccessfulTransactionModal/index.tsx

```diff
@@ -40,17 +42,19 @@ export const SuccessfulTransactionModal: React.FC<ISuccessfulTransactionModalPro
         <h3 css={styles.title}>{title}</h3>
 
         <div css={styles.messageContainer}>
-          <Typography variant="small1" component="p">
-            {message}
-          </Typography>
-
+          {message && (
```

Comment:
Message could be an empty string:
```suggestion
          {!!message && (
```